### DISCUSSION TITLE
Links to scripts in README and web page

### DISCUSSION THREAD

**Discussion by laurentlbm** [tteck/Proxmox#527] [Ideas]
Hi,

As you mention in the README and the webpage, it's important to “[a]lways remember to use due diligence when sourcing scripts and automation tasks from third-party sites.”

Therefore, I think it would be useful if every section contained a link to the actual source script on GitHub, so that users don't need to go searching for it. For example:

> To create a new Proxmox Nginx Proxy Manager LXC Container, run the following in the Proxmox Shell.
>
> ``` 
> bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/nginx-proxy-manager-v4.sh)"
> ```
> [View source](https://github.com/tteck/Proxmox/blob/main/ct/nginx-proxy-manager-v4.sh)

Very nice project btw!

**Comment by tteck** [comment]
Each app uses 3 scripts. For Nginx Proxy Manager:
```
https://github.com/tteck/Proxmox/raw/main/ct/nginx-proxy-manager-v4.sh
```
```
https://github.com/tteck/Proxmox/raw/main/ct/create_lxc.sh
```
```
https://github.com/tteck/Proxmox/raw/main/setup/nginx-proxy-manager-install.sh
```
The latter being the important one. 
I always entertain PR's 😉

**Reply by tteck** [reply]
And, `nginx-proxy-manager-install.sh` calls several other "wget/curl" commands. Those need due diligence also.
